Trace.py

```diff
@@ -326,8 +335,8 @@ def _dump_texture(self, index):
         depth = 1 << depth_shift
 
         self._dbg_print(
-            "Texture %d [0x%08X, %d x %d x %d (pitch: 0x%X), format 0x%X]"
-            % (index, offset, width, height, depth, pitch, fmt_color)
+            "Texture %d [0x%08X, %d x %d x %d (pitch: 0x%X <ignored>), format 0x%X]"
+            % (index, offset, width, height, depth, reg_pitch, fmt_color)
```

Comment:
This must differentiate between swizzled and unswizzled; but usually I'd defer these sort of things to the decoding phase, after capturing.

I'd usually try to only decode what is necessary for capturing (say if you need to download a palette, or know the pixel-size from the format).
You should still log everything, because I found games which even controlled inactive registers as if they were being used - so having this additional data might be useful in some instances, even if it's not relevant for decoding.